src/adapters/aave-v2/products/pool/aaveV2BasePoolAdapter.ts

```diff
@@ -31,6 +33,14 @@ type AaveV2PoolMetadata = Record<
   }
 >
 
+const protocolDataProviderContractAddresses: Partial<Record<Chain, string>> = {
```

Comment:
Same contracts used for metadata, so I've just moved them as a constant for the whole file.